src/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -33,16 +33,14 @@
     <value>Since when is this problem affecting you?</value>
     <language>en_US</language>
     <protected>false</protected>
-    <shortDescription
-        >Since when is this problem affecting you?</shortDescription>
+    <shortDescription>Since when is this problem affecting you?</shortDescription>
   </labels>
```

Comment:
Didn't find the label but I would recommend changing the "Explain the problem" label to "Explain the issue" to align with the changes I'm proposing.